docker/Dockerfile

```diff
@@ -1,24 +1,27 @@
 ARG VERSION
 
-FROM ghcr.io/deephaven/server:${VERSION}
+FROM ghcr.io/deephaven/server-sklearn:${VERSION}
 
 ARG DH_IB_VERSION
 ARG DH_IB_BRANCH
 
+COPY deephaven_ib-0.2.4-py3-none-any.whl /tmp/
+
 # Install requirements
 RUN apt update && \
-    apt install --yes git python3-venv && \
-    python3 -m pip install --upgrade pip && \
-    python3 -m pip install --upgrade setuptools wheel build twine
-
-# Invalidate the cache below here, so that the latest repo changes get cloned
-ARG CACHEBUST
+    apt install --yes git python3-venv zip && \
+    python3 -m pip install --upgrade pip
 
-# Build and install deephaven-ib
 RUN mkdir /build && \
     cd /build && \
-    git clone --branch ${DH_IB_BRANCH} https://github.com/deephaven-examples/deephaven-ib.git && \
-    cd deephaven-ib && \
-    python3 -m build && \
-    pip3 install dist/*.whl && \
-    rm -rf /build
+    curl -o ./api.zip "https://interactivebrokers.github.io/downloads/twsapi_macunix.1016.01.zip" && \ 
```

Comment:
1) tws version should be an ARG.
2) I'm guessing that the mac version may be ok here, but there is some funkiness.  The docker container is AMD64 Linux.  If the tws download has any compiled python stuff, this is going to break.  It would be better to pull and install the linux zip file to protect against IB deciding to deploy a binary python lib.

Reply:
I have addressed this in the latest main.